### DISCUSSION TITLE
Thanks for your plugin

### DISCUSSION THREAD

**Discussion by felixchenier** [Cornerback24/Blender-NLA-Midi-Copier#7] [Show and tell]
Hi there

I just wanted to thank you for this very useful plugin. I managed to do my first Blender animation that is an octopuss who plays several instruments. One of my goals in learning Blender was to put some visuals on songs I write as a hobby, and I see your plugin as a very, very useful tool for reaching this goal.

Here is the video if you're curious: https://www.youtube.com/watch?v=0BG8OmAeUyk

Thank you again, I think I'm going to use this plugin a LOT ;-)

Félix

**Comment by Cornerback24** [comment]
That's a neat Octopus animation! I'm glad the plugin is helpful.
